### DISCUSSION TITLE
Maybe an error?

### DISCUSSION THREAD

**Discussion by SEANIRLO** [AI4Finance-Foundation/FinRL#478] [Q&A] [answered]
I noticed that the function _initiate_state(self) in env_stocktrading.py seems to return the price of the last day of the document instead of the first day. I wonder whether it's a mistake.

**Comment by Athe-kunal** [comment, marked as answer]
This is helpful if you have an online model and you are training on a weekly or monthly basis. Here you don't need to go to the initial state, rather continue with the previous state.
